examples/gpio/gpio_config.c

```diff
@@ -227,15 +249,179 @@ int TPM2_GPIO_Config_Example(void* userCtx, int argc, char *argv[])
 
     /* GPIO configured as an input, requires an extra configuration step */
     if (gpioInput) {
+        BYTE dummy = 0;
         /* Writing a dummy byte has no impact on the input, but is required */
         rc = wolfTPM2_NVWriteAuth(&dev, &nv, nvIndex, &dummy, sizeof(dummy), 0);
         if (rc != 0) {
             printf("Error while configuring the GPIO as an Input.\n");
         }
     }
 
+#elif defined(WOLFTPM_NUVOTON)
+
+    if (caps.mfg != TPM_MFG_NUVOTON) {
+        printf("TPM model mismatch. GPIO support requires a Nuvoton NPCT7xx TPM 2.0 module\n");
+        goto exit;
+    }
+
+    /* GPIO un-configuration is done using NVDelete, no further action needed */
+    /* Nuvoton can reconfigure any GPIO without deleting the created NV index */
+    if (gpioMode == TPM_GPIO_MODE_UNCONFIG) {
+        printf("Deleting GPIO NV Index\n");
+
+        /* This procedure requires CommandCode policy and EK Auth policy */
+        rc = wolfTPM2_StartSession(&dev, &tpmSessionIndex, NULL, NULL,
+                                   TPM_SE_POLICY, TPM_ALG_NULL);
+        if (rc == TPM_RC_SUCCESS) {
+            printf("index ok\n");
+        }
+
+        rc = wolfTPM2_StartSession(&dev, &tpmSessionPlatform, NULL, NULL,
+                                   TPM_SE_POLICY, TPM_ALG_NULL);
+
+        if (rc == TPM_RC_SUCCESS) {
```

Comment:
Is there a reason to not `goto exit` here on error? Same on previous check for startsession?